### DISCUSSION TITLE
Anchoring the version when moving repository

### DISCUSSION THREAD

**Discussion by louis030195** [semantic-release/semantic-release#2768] [Q&A]
Hi, I moved my repo https://github.com/different-ai/embedbase-js recently to 
https://github.com/different-ai/embedbase/tree/main/sdk/embedbase-js
and it seems to have resetted to 1.0.0 the version

https://github.com/different-ai/embedbase/actions/runs/4733340971/jobs/8400727688

(fyi i want to have the github release tag like js-${version} as it is now a monorepo with other things published there)

do you know how i can "anchor" to the old version (1.7.9) ?

**Comment by Tlepel** [comment]
You can probably add a git tag to the commit that should have version 1.7.9, and add a git note using the right format. Check out this page in the documentation ([Semantic Release - release not found release branch after git push --force](https://semantic-release.gitbook.io/semantic-release/support/troubleshooting#release-not-found-release-branch-after-git-push-force)), if you start at step 2 you should have the behaviour you want.

**Comment by jcandan** [comment]
@Tlepel, you nailed it with that link. For clarification, I just needed to grab the remote notes first, then push them to the new repo.

```
git fetch --force old 'refs/notes/*:refs/notes/*'
git notes --ref semantic-release list
git notes --ref semantic-release show 1.0.0-alpha.22
git push origin refs/notes/semantic-release 
```
